src/composables/maxAmount.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  ref,
+  computed,
+  onMounted,
+  watch,
+} from '@vue/composition-api';
+import { pick } from 'lodash-es';
+import BigNumber from 'bignumber.js';
+import { Store } from 'vuex';
+import FUNGIBLE_TOKEN_CONTRACT from 'aeternity-fungible-token/FungibleTokenFullInterface.aes';
+import { TxBuilder, SCHEMA } from '@aeternity/aepp-sdk';
+import { MAGNITUDE, STUB_CONTRACT_ADDRESS, AETERNITY_CONTRACT_ID } from '../popup/utils/constants';
+import { rxJsObservableToVueState } from '../popup/utils/utils';
+
+import {
+  executeAndSetInterval,
+  calculateFee,
+  validateTipUrl,
+  checkAensName,
+  handleUnknownError,
+  watchUntilTruthy,
+} from '../popup/utils';
+import { IAccount } from '../types';
+
+export interface UseMaxAmountOptions {
+  store: Store<any>
```

Comment:
Same as with the router - there is no need to pass the store as an attribute. It can be imported in this file.